virtManager/delete.py

```diff
@@ -517,6 +507,21 @@ def _delete_vm(self, vm, undefine):
             log.debug("Removing VM '%s'", vm.get_name())
             vm.delete()
 
+    def _delete_disks(self, vm, paths, conn, meter):
+        storage_errors = []
+        if not paths:
+            return
```

Comment:
This `if not paths` conditional can be removed, it's left over from the previous mixed storage vs vm implementation.